src/data/actions/enrollmentsList.js

```diff
@@ -0,0 +1,56 @@
+import EnterpriseDataApiService from '../services/EnterpriseDataApiService';
+import { getPageOptionsFromUrl } from '../../utils';
+import {
+  PAGINATION_REQUEST,
+  PAGINATION_SUCCESS,
+  PAGINATION_FAILURE,
+} from '../constants/table';
+import NewRelicService from '../services/NewRelicService';
+
+const tableId = 'enrollments';
+
+const searchEnrollmentsListRequest = options => ({
+  type: PAGINATION_REQUEST,
+  payload: {
+    tableId,
+    options,
+  },
+});
+const searchEnrollmentsListSuccess = data => ({
+  type: PAGINATION_SUCCESS,
+  payload: {
+    tableId,
+    data,
+  },
+});
+const searchEnrollmentsListFailure = error => ({
+  type: PAGINATION_FAILURE,
+  payload: {
+    tableId,
+    error,
+  },
+});
+
+// This is doing nearly the same thing the table actions do however this
+// is necessary so we can pass in the `search` parameter. We leverage the same
+// events as the table actions, so the same `table` reducers will be called.
+const searchEnrollmentsList = searchOptions => (
```

Comment:
Do we need to create a new action creator function here? Could we use the already existing [`paginateTable`](https://github.com/edx/frontend-app-admin-portal/blob/master/src/data/actions/table.js#L62) action instead? Seems like we could, which would eliminate the need for the new enrollmentsList.js file.

Also, the `getPageOptionsFromUrl` util function updates in this PR below is also used in the `paginateTable` function. The update made in this PR to that util function should be enough to get the search query parameter  from the URL without needing to pass in `searchOptions` here.